test/lib/ufe/testAttribute.py

```diff
@@ -222,7 +222,7 @@ def testAttributeBool(self):
 
         # check to see if the attribute edit is allowed
         isAttrEditAllowed = mayaUsdUfe.isAttributeEditAllowed(usdAttr)
-        self.assertFalse(isAttrEditAllowed, False)
+        self.assertFalse(isAttrEditAllowed)
```

Comment:
Correct, the logic in the "if isAttrEditAllowed:" conditions are dead code. 
When these test were written, we didn't have such restriction to block attribute edits but now we have. In the case of top_layer.ma example, the top layer "Ball.maya.usd" has already set certain attributes ( e.g doubleSided, etc... ) and setting them in the weaker layers are blocked now and therefore the following tests after "if isAttrEditAllowed:" condition results in error.

Reply:
Right --- and my suggestion here, to keep the test code coverage for the UFE attributes support, is to set the attributes in the stronger layer, otherwise we won't be testing the Ufe::Attribute::set() interfaces any more.  You've already added two new tests to test attribute blocking, so we don't need additional testing for that, but we do need to keep the test coverage for Ufe::Attribute::set().

Reply:
Fair enough, let me have a look.